pom_confluent.xml

```diff
@@ -368,7 +368,7 @@
         <dependency>
             <groupId>net.snowflake</groupId>
             <artifactId>snowflake-ingest-sdk</artifactId>
-            <version>1.0.2-beta.4</version>
+            <version>1.0.2-beta.7</version>
```

Comment:
was there some perf regression in 1.0.2-beta.5? I saw some jira from @sfc-gh-lsembera

Reply:
Yes, there was a perf regression relating to timestamps.

Reply:
Yes, but beta.5-beta.7 also contain a bunch of corruption/wrong result fixes, so I think it would be good to upgrade since correctness is more important than performance (for example: https://snowflakecomputing.atlassian.net/browse/SNOW-718273 shows we're hitting it in SFCTEST0)